README.md

```diff
@@ -1,3 +1,78 @@
-# skywalking-swck
-
 Apache SkyWalking Cloud on Kubernetes
+============
+
+![](https://github.com/apache/skywalking-swck/workflows/Build/badge.svg?branch=master)
+
+<img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
+
+A bridge project between [Apache SkyWalking](https://github.com/apache/skywalking) and Kubernetes.
+
+SWCK is a platform for the SkyWalking user, provisions, upgrades, maintains SkyWalking relevant components, and makes them work natively on Kubernetes. 
+
+# Quick Start
+
+ 1. Go to the [download page](https://skywalking.apache.org/downloads/) to download latest release manifest. 
+ 
+ 2. Create the `skywalking-swck-system` namespace:
+ ```
+ kubectl create namespace skywalking-swck-system
+ ```
+ 3. Apply the manifests for the Controller and CRDs in release/config:
+ ```
+ kubectl apply -f release/config
+ ```
+ 
+# Guides of Deployment
+## Use kustomize to customise your deployment
+
+1. Clone the source code:
+
+```sh
+git clone git@github.com:apache/skywalking-swck.git
+```
+
+2. Edit file `config/default/kustomization.yaml` file to change your preferences. If you prefer to your private docker image, a quick path to override `IMG` environment variable : `export IMG=<private registry>/controller:<tag>`
+
+3. Use `make` to generate the final manifests and deploy:
+
+```sh
+make deploy
```

Comment:
Due to network reasons, `gcr.io/kubebuilder/kube-rbac-proxy:v0.4.1` may fail to pull, please add tips.